packages/snaps-sdk/src/jsx/components/Address.ts

```diff
@@ -10,6 +10,9 @@ import { createSnapComponent } from '../component';
  */
```

Comment:
Missing JSDoc

Reply:
Also add it to the `Address` JSDoc underneath

Reply:
Updated both.